src/parts/AccountModal/index.tsx

```diff
@@ -158,6 +161,8 @@ const AccountModal = ({
         <>
           <p>
             {t('install_supported_wallets')}
+            <ExternalLink href={TERMS_AND_CONDITIONS_LINK}>terms and conditions</ExternalLink>
```

Comment:
Minor: was this meant to be hardcoded, or did you intend to use the `nav_terms_and_conditions` dictionary entry (or a different one)? Would be good to keep copy in the dictionary consistently.

Reply:
@tomjeatt 
That makes sense.